### PR TITLE
Popup modal window

### PR DESCRIPTION
**Pull request by prius2055**
### Pop-up modal window

**This is the pop-up modal window for the works section of the project**

- The works section is displayed when the page loads, using Javascript logic
- Clicking on any of the project cards brings up a modal that shows details of the project.

### CODE REVIEW

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi Team👋👋, 

tryagain


Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights✅✅

- [x] Linters are passing👍
- [x] When the user clicks (or taps) the button to check project details, the popup with details about the project appears👍.
- [x] When the user clicks (or taps) the close (X) button, the popup disappears👍.
- [x] Github flow practice has been used👍.


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- See comments below😎

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -118,7 +125,7 @@ <h2>Multi-Post Stories</h2>
           </div>
 
           <div class="works-card">
-            <div class="work-card">
+            <!-- <div class="work-card">
```
[the remark this answers is not in the document]
Comment (prius2055) [reply index.html:128]: Thank you. This has been fixed

[the remark this answers is not in the document]
Comment (prius2055) [reply index.html:128]: Thank you, I have fixed this

**Review by Rustamxon7 — APPROVED** [review]
Hi @prius2055, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Developer-512px-10

## Highlights
1. No Linters errors 💯
2. All requirements are complete 💯
3. Gerat readme documentation 💯 

## Optional suggestions
No optional suggestions :)

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**